librz/bin/format/le/le.c

```diff
@@ -94,6 +94,28 @@ static RzBinSymbol *le_get_symbol(rz_bin_le_obj_t *bin, ut64 *offset) {
 	return sym;
 }
 
+static bool read_le_entry_bundle_entry(RzBuffer *buf, ut64 offset, LE_entry_bundle_entry *e, LE_entry_bundle_type type) {
+	switch (type) {
+	case ENTRY16:
+		return rz_buf_read8_offset(buf, &offset, &e->entry_16.flags) &&
```

Comment:
I am not sure about taking the address of the argument variable. What if it becomes a register on some architectures?

Reply:
good point.

Reply:
Don't forget this one please